### PR TITLE
Fix/with-nestjs-example

### PR DESCRIPTION
**Pull request by spotlesscoder**
project was not building successfully
 - @nestjs/config missing
 - ExpressAdapter type was incompatible to the used bull board version

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (felixmosh) [inline .devcontainer/devcontainer.json:1]: Please remove this file from this PR

File: examples/with-nestjs/src/queues/basic-auth.middleware.ts [hunk 2]
```diff
@@ -30,7 +31,10 @@ export class BasicAuthMiddleware implements NestMiddleware {
       return;
     }
 
-    const isPasswordValid = await await bcrypt.compare(password, this.passwordHash);
+    const isPasswordValid = await await bcrypt.compare(
```
Comment (felixmosh) [inline examples/with-nestjs/src/queues/basic-auth.middleware.ts:34]: why there is 2 `await` here?

**Review by felixmosh — CHANGES_REQUESTED** [review]

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (spotlesscoder) [reply .devcontainer/devcontainer.json:1]: done

File: examples/with-nestjs/src/queues/basic-auth.middleware.ts (hunk 2 above)
Comment (spotlesscoder) [reply examples/with-nestjs/src/queues/basic-auth.middleware.ts:34]: maybe I forgot to save before commit. will push a fix

**Comment by felixmosh** [comment]
Thank you for you contribution  🙏🏽

**Comment by spotlesscoder** [comment]
you're welcome :) thank you too for making this tool - it helps me keeping track of things :)

**Pull request merged**